packages/sku/lib/runPrettier.js

```diff
@@ -36,22 +37,19 @@ const runPrettier = async ({ write, listDifferent, paths }) => {
   const pathsToCheck = paths && paths.length > 0 ? paths : ['.'];
 
   prettierArgs.push(...pathsToCheck);
-  /*
-   * Show Prettier output with stdio: inherit
-   * The child process will use the parent process's stdin/stdout/stderr
-   * See https://nodejs.org/api/child_process.html#child_process_options_stdio
-   */
-  const processOptions = {
```

Comment:
Chose to just inline this object because it removes the need to the JSDoc equivalent of `as const`.